app/src/components/Project/Content/AllCardsContent.tsx

```diff
@@ -140,6 +161,149 @@ export default function AllCardsContent({
           </div>
         )}
       </div>
+
+      {/* Modal */}
+      {/* <Modal
+        opened={showModal}
+        onClose={() => setShowModal(false)}
+        withCloseButton={true}
+        title='Project settings'
+        size='60%'
+        padding={0}
+        overlayProps={{
+          backgroundOpacity: 0.6,
+        
+        }}
+      > */}
+
+      <Modal
+        opened={showModal}
+        onClose={() => setShowModal(false)}
+        withCloseButton={true}
+        title='Project settings'
+        size='60%'
+        padding={0}
+        styles={{
+          inner: {
+            overflow: 'hidden',
+          },
+          body: {
+            overflow: 'hidden',
+            maxHeight: 'calc(100vh - 150px)', // Adjust the height to fit within the screen
+            display: 'flex',
+            flexDirection: 'column',
+          },
+        }}
+        overlayProps={{
+          backgroundOpacity: 0.6,
+        }}
+      >
+        <div className='h-[800px] flex flex-col'>
+          <div onClick={toggleText} className='font-medium cursor-pointer m-3 text-xl font-bold'>
+            EROS
+          </div>
+          <div className={clsx(showTextEros ? 'block' : 'hidden', 'ml-3')}>
+            The Enterprise Resource Optimization System (EROS) is a cutting-edge IT project designed
+            to revolutionize the way our organization manages and utilizes its resources. EROS
+            integrates advanced technologies and smart algorithms to streamline and optimize key
+            business processes, ensuring maximum efficiency and resource utilization across
+            departments.{' '}
```

Comment:
@Royamosavipour this text is supposed to come from project description, we don't need it hardcoded here like this.

@eugeek  FYI

Reply:
Thank you for your guidance